game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyServerProperties.java

```diff
@@ -29,10 +30,10 @@
   @Nonnull private final String host;
 
   /** The port the lobby is listening on. */
-  private final int port;
+  @Nonnull private final Integer port;
```

Comment:
Did I miss something, or why is this a non-null Integer instead of an int again?

Reply:
See https://github.com/triplea-game/triplea/pull/4520#discussion_r245491990.

I made a mistake when converting this to `int`.  Currently, the only way you can tell the Lombok builder that a field is required is though the use of `@Nonnull`, and obviously that must be applied to a reference type.  For a primitive `int`, if the user doesn't specify the port while building, it just gets set to zero.  While for a `@Nonnull Integer`, if it's not specified, an unchecked exception will be thrown at runtime.